bindings/wasm/src/xmtpv3.ts

```diff
@@ -134,6 +140,15 @@ export class XMTPWasm {
     return decrypt_message(handle, sessionId, ciphertext);
   }
 
+  getPublicAccountJSON(handle: string): string {
+    return get_public_account_json(handle);
+  }
+
+  addOrGetPublicAccountFromJSON(json: string): VoodooInstance {
+    const handle = add_or_get_public_account_from_json(json);
+    return new VoodooInstance(this, handle);
+  }
```

Comment:
I'd expect the return types to match given the names of the functions

Reply:
Like it should be returning a handle? It's more ergonomic if it returns a VoodooInstance but I can rename it

Reply:
I added a comment to explain that it returns a VoodooInstance for now, until we have a corresponding public type